tf2onnx/tfonnx.py

```diff
@@ -700,6 +699,14 @@ def process_tf_graph(tf_graph, continue_on_error=False, verbose=False, target=No
                        "please upgrade onnx package to avoid potential conversion issue.",
                        utils.get_onnx_version(), opset)
 
+    if shape_override:
+        for name, shape in shape_override.items():
+            tf_graph.get_tensor_by_name(name).set_shape(shape)
+
+    tf_graph = infer_shape_for_graph(tf_graph)
+    # FIXME: remove it if no long supporting TF 1.5
+    tf_graph = infer_shape_for_graph_supplementary(tf_graph)
```

Comment:
i would prefer to have a single call here.
maybe ```infer_shape(tf_graph, shape_override)```?

Reply:
need to reload graph after apply shape_override? even all shapes are there after shape override.
will a wrongly specified shape_override break tf shape inference?